v3/cypress/e2e/axis.spec.ts

```diff
@@ -280,6 +280,16 @@ context("Test graph axes with various attribute types", () => {
     ah.removeAttributeFromAxis(arrayOfAttributes[7], "top")
     cy.get("[data-testid=graph]").find("[data-testid=axis-bottom]").find(".sub-axis-wrapper").should("have.length", 1)
   })
+  it("will create a graph with categorical x-axis, categorical y2-axis, and numerical y-axis", () => {
```

Comment:
```suggestion
  it("will create a graph with categorical x-axis, categorical right-axis, and numerical y-axis", () => {
```
The `y2` terminology is only used when the right axis is numeric.